src/DearImGui/SDL/OpenGL.hs

```diff
@@ -49,9 +49,9 @@ Cpp.using "namespace ImGui"
 
 
 -- | Wraps @ImGui_ImplSDL2_InitForOpenGL@.
-sdl2InitForOpenGL :: MonadIO m => Window -> GLContext -> m ()
-sdl2InitForOpenGL (Window windowPtr) glContext = liftIO do
-  [C.exp| void { ImGui_ImplSDL2_InitForOpenGL((SDL_Window*)$(void* windowPtr), $(void* glContextPtr)); } |]
+sdl2InitForOpenGL :: MonadIO m => Window -> GLContext -> m Bool
+sdl2InitForOpenGL (Window windowPtr) glContext = ( 0 /= ) <$> liftIO do
+  [C.exp| bool { ImGui_ImplSDL2_InitForOpenGL((SDL_Window*)$(void* windowPtr), $(void* glContextPtr)) } |]
```

Comment:
I like to keep the `liftIO` right at the top, so for consistency:

```suggestion
sdl2InitForOpenGL (Window windowPtr) glContext = liftIO do
  ( 0 /= ) <$> [C.exp| bool { ImGui_ImplSDL2_InitForOpenGL((SDL_Window*)$(void* windowPtr), $(void* glContextPtr)) } |]
```

Reply:
OK